res/res_pjsip_endpoint_identifier_ip.c

```diff
@@ -187,6 +187,31 @@ static int header_identify_match_check(void *obj, void *arg, int flags)
 		return 0;
 	}
 
+	/* Match match_header against request uri as well */
+	if (!strcasecmp(identify->match_header_name, "request")) {
```

Comment:
While unlikely this could alter behavior for existing users.

Reply:
Indeed, if the user has a match_header for "request" (case insensitive). However custom headers should start with X- or P- as far as I know. But it doesn't matter much, since you requested a separate identify type for this in your next comment, so that will solve this one as well.